rpm/native_db.go

```diff
@@ -393,14 +394,12 @@ func constructEVR(b *strings.Builder, info *Info) string {
 	return b.String()
 }
 
-func constructHint(b *strings.Builder, info *Info) string {
-	b.Reset()
+func constructHint(_ *strings.Builder, info *Info) string {
```

Comment:
this may be mentioned elsewhere, but if we are removing the need of the builder, then why keep the parameter around?

Reply:
To minimize the code churn in this PR. #869 cleans this up when reworking the `rpm` package, so I didn't bother pulling that change "forward."